extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/SmallRyeReactiveMessagingProcessor.java

```diff
@@ -134,12 +135,17 @@ public boolean appliesTo(AnnotationTarget.Kind kind) {
                 return kind == org.jboss.jandex.AnnotationTarget.Kind.CLASS;
             }
 
+            @Override
+            public int priority() {
+                return 500;
```

Comment:
It might make sense to add a comment that the value is `500` because the Spring DI's annotation transformer has the default priority of value `1000` and we want it to take precedence?

Also I would squash the changes, i.e. remove the first commit completely...